api/envoy/config/common/matcher/v3/action.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package envoy.config.common.matcher.v3;
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.common.matcher.v3";
+option java_outer_classname = "ActionProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Common Match Actions]
+
+// Indicates that the associated action should be skipped.
+message SkipAction {
```

Comment:
Do we anticipate that this will be useful for types other than filters?  For example, it's not clear to me that this would ever be used in (e.g.) SSL SAN matching or in the RouteConfiguration.

If there are actually other use-cases where this will be useful, then I'm fine putting it here.  But if it's actually specific to filters, then maybe it should go with the filter or extension protos instead of being in this directory.

Reply:
That's true, I think this primarily applies to filters. I'm not sure if we have any shared proto directories for all kinds of filters though (e.g. both network and HTTP). I'll see if I can find a better place